api/python/requirements/install.txt

```diff
@@ -0,0 +1,5 @@
+salt == 3002.2
```

Comment:
Can we set this as base version?
```salt >= 3002.2```

Reply:
I believe we discussed that before: better to avid automatic deps shifts (especially for salt) since it may break the logic. If want to update the version of salt, better to test that and update the references manually. So it would be under control.